docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,13 @@
+Iris 1.9 Bugs Fixed
+===================
+Iris 1.9 Features
+=================
+Iris 1.9 Incompatible Changes
+=============================
+Iris 1.9 Deprecations
+=====================
+Iris 1.9 Documentation Changes
+==============================
+* Added a first pass of an aggregator to build a "What's New" document from submitted RST files for each change.
+ * Added a new page to the dev docs to explain the format for an entry.
+ * Added a reference to the new page, to the Dev Docs index.
```

Comment:
I suggest that this file is not under source control
it should only be provided at the point where it has been edited